### PR TITLE
feat: add server logs endpoint

### PR DESCRIPTION
**Pull request by AngelOnFira**
This will close RVTEE-313, RVTEE-310, RVTEE-163, RVTEE-311, RVTEE-308, RVTEE-307, RVTEE-306

### CODE REVIEW

**Comment by AngelOnFira** [comment]
* **#1005** <a href="https://app.graphite.dev/github/pr/rivet-gg/rivet/1005?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @AngelOnFira and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: lib/job-runner/src/main.rs [hunk 1]
```diff
@@ -28,6 +28,7 @@ fn main() -> anyhow::Result<()> {
 	let root_user_enabled = std::env::var("NOMAD_META_root_user_enabled")
 		.context("NOMAD_META_root_user_enabled")?
 		== "1";
+	let runner = std::env::var("NOMAD_META_runner").unwrap_or("job_run".to_string());
```
Comment (AngelOnFira) [inline lib/job-runner/src/main.rs:31]: This should work for any existing usage of job-runner, while the new dynamic servers passes in a new var.

File: svc/api/servers/src/assert.rs [hunk 2]
```diff
@@ -61,3 +61,21 @@ pub async fn namespace_for_game(
...
+pub async fn server_for_game(ctx: &Ctx<Auth>, server_id: Uuid, game_id: Uuid) -> GlobalResult<()> {
+	let server_get_res = op!([ctx] ds_server_get {
+		server_ids: vec![server_id.into()],
+	})
+	.await?;
+
+	// let server_data = unwrap!(server_get_res.servers.first());
+	// let server_game_id = unwrap_ref!(server_data.game_id).as_uuid();
+
+	// ensure_eq!(game_id, server_game_id, "server does not belong to game");
```
Comment (AngelOnFira) [inline svc/api/servers/src/assert.rs:78]: This isn't correct rn, fixing all auth upstack after this is merged

File: fern/definition/servers/logs.yml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+imports:
+  commons: ../common.yml
+  uploadCommons: ../upload/common.yml
+  cloudCommons: ../cloud/common.yml
+
+service:
+  auth: true
+  base-path: /servers
+  endpoints:
+    listBuilds:
```
Comment (MasterPtato) [inline fern/definition/servers/logs.yml:12]: Should this be called `listBuilds`?

File: svc/pkg/ds-log/ops/read/tests/integration.rs [hunk 4]
```diff
@@ -0,0 +1,89 @@
+// use chirp_worker::prelude::*;
```
Comment (MasterPtato) [inline svc/pkg/ds-log/ops/read/tests/integration.rs:1]: Should these tests be reimplemented?

**Review by MasterPtato — CHANGES_REQUESTED** [review]

File: svc/pkg/ds-log/ops/read/tests/integration.rs (hunk 4 above)
Comment (NathanFlurry) [reply svc/pkg/ds-log/ops/read/tests/integration.rs:1]: i'm going to get as much as i can up to speed soon

**Pull request merged**